Tests/UnitTests/Plugins/Sycl/Seeding/CommandLineArguments.h

```diff
@@ -0,0 +1,21 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+#include <fstream>
+
+struct CommandLineArguments {
+  void parse(int argc, char** argv);
+  bool allgroup = false;
+  bool onlyGpu = false;
+  bool matches = false;
+  uint groups = 500;
```

Comment:
I don't like this type... You should just use the C\+\+ standard `unsigned int` type instead.